### PR TITLE
[Bug][DEVOPS][SIMS-#1493]-FORMS_URL

### PR DESCRIPTION
**Pull request by guru-aot**
FORMS_URL removed from secrets and assigning from the forms url during formio server creation.

### CODE REVIEW

File: devops/openshift/forms-deploy.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ objects:
       FORMIO_ROOT_PASSWORD: "${FORMIO_ROOT_PASSWORD}"
       MONGODB_URI: "${MONGODB_URI}"
       NODE_CONFIG: '{"mongo": "${MONGODB_URI}"}'
+      FORMS_URL: 'https://${FORMS_URL}'
```
Comment (andrewsignori-aot) [inline devops/openshift/forms-deploy.yml:16]: Can the `https` be part of `FORMS_URL`?

Comment (guru-aot) [reply devops/openshift/forms-deploy.yml:16]: i feel it cannot be done, else when forms is created the route needs just the domain. Else we need to remove the https, on all the place we try to create the forms.

File: devops/openshift/api-deploy.yml [hunk 2]
```diff
@@ -98,12 +96,17 @@ objects:
                 - name: FORMS_SA_USER_NAME
                   valueFrom:
                     secretKeyRef:
-                      key: ${FORMS_SA_USER_NAME_NAME_KEY}
+                      key: ${FORMIO_ROOT_EMAIL_NAME_KEY}
```
Comment (ann-aot) [inline devops/openshift/api-deploy.yml:99]: what is this change?

Comment (ann-aot) [reply devops/openshift/api-deploy.yml:99]: nevermind. i got it

**Review by ann-aot — APPROVED** [review]
👍

**Review by andrepestana-aot — APPROVED** [review]
Looks good!

**Review by andrewsignori-aot — APPROVED** [review]
👍

**Pull request merged**